### ISSUE TITLE
3.4.3 serviceResolved not called

### ISSUE THREAD

**Issue by Idhor** [jmdns/jmdns#50]
I have just tried JmDNS 3.4.3 on a network with airplay and chromecast devices
I have noticed that serviceResolved is not called anymore
On the logcat i see those kind of warnings:
```
02-23 09:54:42.205 1764-1915/my.package W/j.j.i.DNSIncoming: Could not find record class. domain: local. type: TYPE_IGNORE index 0
                                                                              dns[response,192.168.1.38:5353, length=125, id=0x0, flags=0x8400:r:aa, answers=1
                                                                              answers:
                                                                              	[Service@172513216 type: TYPE_SRV index 33, class: CLASS_IN index 1-unique, name: Chromecast5471._googlecast._tcp.local. ttl: '119/120' server: 'Chromecast5471:8009']]
                                                                              	answer:        [Service@172513216 type: TYPE_SRV index 33, class: CLASS_IN index 1-unique, name: Chromecast5471._googlecast._tcp.local. ttl: '119/120' server: 'Chromecast5471:8009']
                                                                                 0: 0000840000000001 000000010e436872 6f6d656361737435 3437310b5f676f6f     ........ .....Chr omecast5 471._goo
                                                                                20: 676c656361737404 5f746370056c6f63 616c000021800100 000078001c000000     glecast. _tcp.loc al..!... ..x.....
                                                                                40: 001f490e4368726f 6d65636173743534 3731056c6f63616c 000e4368726f6d65     ..I.Chro mecast54 71.local ..Chrome
                                                                                60: 6361737435343731 056c6f63616c0000 0180010000007800 04c0a80126           cast5471 .local.. ......x. ....&
02-23 09:54:44.956 1764-1915/my.package W/j.j.i.c.DNSRecordType: Could not find record type for index: 49220
```

**Comment by kaikreuzer** [comment]
Was it any different with 3.4.2?

**Comment by Idhor** [comment]
I have just changed the version number on the build.grade (with the package rename)
If i downgrade back to 3.4.2 it works

**Comment by Idhor** [comment]
@kaikreuzer i think that i have found the offending change
I have done some tests and it seems that is all related to the change of `USE_DOMAIN_NAME_FORMAT_FOR_SRV_TARGET` on the commit of sl4fj https://github.com/jmdns/jmdns/commit/5a8489186254d49141315195644361731007d25c#diff-280355df0b37afdb4aa6561c06374fc1L33

**Comment by kaikreuzer** [comment]
Argh, sh$%&t, this should indeed never have been changed in there!
Ok, I suggest to revoke 3.4.3, fix it and re-release it.

**Comment by kaikreuzer** [comment]
Could you please have a try with https://bintray.com/artifact/download/jmdns/mvn/org/jmdns/jmdns/3.4.3/jmdns-3.4.3.jar and let me know if it fixes the problem?

**Comment by kaikreuzer** [comment]
Sorry, there is still something wrong about the jar... Let me try again :-/

**Comment by Idhor** [comment]
ok, meanwhile i have found another issue regarding serviceResolved (it happens on 3.4.2 too) i will open a separate issue for it

**Comment by kaikreuzer** [comment]
Ok, I have revoked the 3.4.3 release and I will actually better do a 3.5.0 instead as a replacement (feeling that the version fits better to the fact that a new dependency has been introduced).

>  i will open a separate issue for it

If you could directly come up with a fix and a PR, I would also be fine to integrate it directly in 3.5.0.

**Comment by kaikreuzer** [comment]
Fixed with https://github.com/jmdns/jmdns/commit/cbc80c83fab69008debcad875323804f9ce3aaaf
